src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php

```diff
@@ -383,4 +383,22 @@ protected function isCsrfTokenValid(string $id, string $token): bool
 
         return $this->container->get('security.csrf.token_manager')->isTokenValid(new CsrfToken($id, $token));
     }
+
+    /**
+     * Dispatches a message to the bus.
+     *
+     * @param object $message The message to dispatch
+     *
+     * @return mixed The result from the bus
+     *
+     * @final since version 4.1
+     */
+    protected function dispatch($message)
```

Comment:
Small concern about the naming, we already have an event-dispatcher component... so this could take an event. What about `broadcast`? It's quite similar, I'm just trying to limit confusions, fine with it as is.

Reply:
Please forget about it :) messages could very well be events.